code/__HELPERS/icons.dm

```diff
@@ -1167,4 +1167,14 @@ world
 		dummy_key ? unset_busy_human_dummy(dummy_key) : qdel(body)
 		return out_icon
 	else
-		return humanoid_icon_cache[icon_id]
\ No newline at end of file
+		return humanoid_icon_cache[icon_id]
+
+
+/proc/Get_Pixel_Angle(y, x)
+	if(!y)
+		return (x >= 0) ? 90 : 270
+	. = arctan(x / y)
```

Comment:
beware byond trig functions have errors, see shuttle code for less buggy angle calc

Reply:
Which one should I use?

Reply:
Also this is just for a visual effect so any errors don't matter that much